.environment

```diff
@@ -1,22 +1,132 @@
-# Initial admin user on first deploy.
-export INIT_ADMINUSER='admin@example.com'
-export INIT_ADMINPW='password'
+####################################################################################################
+# General
 
-# Public url for the current Platform.sh environment.
-export PUBLIC_URL=$(echo $PLATFORM_ROUTES | base64 --decode | jq -r 'to_entries[] | select(.value.primary == true) | .key')
+# Public url for the current Platform.sh environment. Remove trailing slash.
+ENVIRONMENT=$(echo $PLATFORM_ROUTES | base64 --decode | jq -r 'to_entries[] | select(.value.primary == true) | .key')
+export PUBLIC_URL=${ENVIRONMENT%/}
+export LOG_STYLE="pretty"
+if [ "$PLATFORM_BRANCH" != "master" ] ; then
+    export LOG_LEVEL="silent"
+else
+    export LOG_LEVEL="silent"
```

Comment:
@chadwcarlson what is the difference between lines 9 and 11?
Both set the `LOG_LEVEL="silent"`